packaging/ubuntu/rules

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/make -f
+
+export DH_VERBOSE = 1
+
+.ONESHELL:
+include /etc/os-release
+
+STATIC := 0 # can't use golang tracee-rules if enabled
+BTFHUB := 1 # will include tailored BTF files from btfhub
+
+TRACEE_MAKE = BTFHUB=$(BTFHUB) STATIC=$(STATIC) make -f Makefile.one
```

Comment:
For all available options (versions of ubuntu) tracee-ebpf will be built with CO-RE and BTFHUB support, correct?

Reply:
Yes it is always CORE, non static and with BTFHUB support. Shared libs according to the environment it was built on.